examples/manual/share.fmf

```diff
@@ -0,0 +1,56 @@
+summary: A set of manual tests with common setup and cleanup
+
+description:
+    Demonstrates how fmf inheritance can be used to prevent
+    unnecessary duplication of setup and cleanup details.
+
+manual:
+    setup:
+        Install latest tmt package from the copr repository.
```

Comment:
Could we treat this as a bash script, if we want to run this in semi-automated way later on?
```suggestion
        # Install latest tmt package from the copr repository.
```

Reply:
Interesting idea. Yes, I think for semi-automated workflows something like that could be useful.

Reply:
Updated the example with your suggestion.